### ISSUE TITLE
3x2 grid for drag window to tile 

### ISSUE THREAD

**Issue by lodriguez** [lucmos/UltrawideWindows#16]
Well, right now we only have shortcuts to tile windows. It would be great if we could also use the "drag to tile" option with the new grid as well.

**Comment by lucmos** [comment]
I agree that would be great. Do you have any pointer to docs or examples on how to implement it?

**Comment by lodriguez** [comment]
Not yet. I haven't done anything with the "KDE project" yet. I think I will have a look at "kwinscreenedges", that might be a good starting point.

**Comment by sdfg2** [comment]
I came here to see if this was possible.  Windows PowerToys has FancyZones which is extremely powerful - hold shift and drag a window to a tile (within a user-defined layout per display) and it will fill that tile.

**Comment by Stijn-vdD** [comment]
Check this project out: https://github.com/1mikegrn/snappyzones
I have not personally tried it out but it's supposed to provide the functionality of FancyZones on Linux.
